### DISCUSSION TITLE
Can't import DeepAREstimator from `gluonts.model`

### DISCUSSION THREAD

**Discussion by ArianKhorasani** [awslabs/gluonts#2993] [Q&A] [answered]
I did install GluonTS on a cluster via `pip install gluonts` but I'm unable to use from `_gluonts.model.deepar import DeepAREstimator_ ` and _`gluonts.model.transformer import TransformerEstimator`_ since it is not defined, getting ModuleNotFoundError. I don't know why I can't install the whole models in gluonts via pip install gluonts command line. I'm using python 3.8.2 version on Macbook, but as mentioned above, I'm installing ` _gluonts_` on a Cluster in virtual machine.

Any help would be appreciated @lostella et al :)

**Comment by lostella** [comment, marked as answer]
```
from gluonts.mx.model.deepar import DeepAREstimator
```

for the mxnet-based version, and 

```
from gluonts.torch.model.deepar import DeepAREstimator
```

for the torch-based version.

**Comment by lostella** [comment]
@ArianKhorasani does that do it?

**Comment by ArianKhorasani** [comment]
> @ArianKhorasani does that do it?

Yes, @lostella! Thanks for your help! for DeepAREstimator, it's working as you mentioned, but I'm facing issues while using TransformerEstimator, I'm using from `gluonts.model.transformer import TransformerEstimator` but getting error that It is not defined. please note that I have already installed gluonts for torch and mxnet! 

I did try` gluonts.mx.model.transformer import TransformerEstimator` which works fine, but I was looking for torch models and using TransformerModel there too! 

Appreciate your suggestions!

**Comment by lostella** [comment]
Right: not all models are implemented also in torch, this table has it summarized https://ts.gluon.ai/stable/getting_started/models.html

**Comment by ArianKhorasani** [comment]
> Right: not all models are implemented also in torch, this table has it summarized https://ts.gluon.ai/stable/getting_started/models.html

Thanks a lot @lostella for the link and time! Please mark this issue as resolved! If I had another question, will open an issue if you wouldn't mind ;)
